### ISSUE TITLE
Suggestion: add support for compute shaders

### ISSUE THREAD

**Issue by mroowaz** [markusfisch/ShaderEditor#38]
Is there any chance to add support for OpenGL ES 3.1 compute shaders?

Best regards,
MroowaZ

**Comment by markusfisch** [comment]
Of course, there's always a chance! This is open source :) And compute shaders would be really interesting I think. I'd expect some hassles getting them to work where OpenGL ES 3.1 is available without breaking for devices with OpenGL ES 2 but in theory in should be possible.

So it's not on the list yet but I wouldn't rule it out for the future.
